charts/dynamic-link-server/values.yaml

```diff
@@ -0,0 +1,9 @@
+---
+image:
+  repository: ghcr.io/neuralegion/dynamic-link-server
+  tag: latest
+  pullPolicy: Always
+ingress:
+  url: dls-1q2w3e.k3s.brokencrystals.nexploit.app
```

Comment:
If we can delete the default value, it'll be better

Reply:
Fixed it